### PR TITLE
BUGFIX: BB-2122 primary measure missing error fix

### PR DESCRIPTION
**Pull request by strouhaljan**
---

[Check PR owner responsibilities](https://confluence.intgdc.com/display/Development/Code-reviews#Code-reviews-Ownerresponsibilities)

Supported PR commands:

Command | Description
--- | ---
`extended test - examples` | Live examples tests
`extended test - storybook` | Storybook screenshot tests

See more [options](https://confluence.intgdc.com/display/kbhr/How+to+work+with+git+and+Github#HowtoworkwithgitandGithub-Extendedchecks).

# PR Checklist

- [ ] Verify code changes ([Checklist](https://confluence.intgdc.com/display/Development/Code-reviews+checklist), [Best practices](https://confluence.intgdc.com/display/Development/Code-reviews+best+practices))
- [ ] [Verify pull-request formalities](https://confluence.intgdc.com/display/Development/Code-reviews)
- [ ] Change was tested by using [gdc-dev-release](https://confluence.intgdc.com/display/~tomas.vojtasek/Private+NPM) in [gdc-analytical-designer](https://github.com/gooddata/gdc-analytical-designer) and [gdc-dashboards](https://github.com/gooddata/gdc-dashboards) (if applicable)
- [ ] Migration guide (for major changes) is mentioned in [CHANGELOG.md](../blob/master/CHANGELOG.md).
- [ ] Successful `extended test - examples`
- [ ] Successful `extended test - storybook`
- [ ] Checked yarn.lock consistency (no dep. duplicities especially Goodstrap)

# Related PRs

- gdc-analytical-designer:
- gdc-dashboards:

# Related Jira tasks

### CODE REVIEW

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-unit-tests-zuul-docker-el7/6231/ : SUCCESS in 5m 31s
- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-oss-check/1923/ : SUCCESS in 4s

File: src/internal/utils/uiConfigHelpers/bulletChartUiConfigHelper.ts [hunk 1]
```diff
@@ -47,7 +47,7 @@ export function getBulletChartUiConfig(
 
     const primaryMeasuresCount = getItemsCount(buckets, BucketNames.MEASURES);
     const secondaryMeasuresCount = getItemsCount(buckets, BucketNames.SECONDARY_MEASURES);
-    const tertiaryMeasuresCount = getItemsCount(buckets, BucketNames.SECONDARY_MEASURES);
+    const tertiaryMeasuresCount = getItemsCount(buckets, BucketNames.TERTIARY_MEASURES);
```
Comment (xMort) [inline src/internal/utils/uiConfigHelpers/bulletChartUiConfigHelper.ts]: 😂

**Comment by xMort** [comment]
LGTM

**Comment by gdgate** [comment]
Build failed (gate pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-storybook-zuul-docker/3233/ : SUCCESS in 17m 39s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-unit-tests-zuul-docker-el7/6235/ : SUCCESS in 17m 57s
- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-merge-compliance-check/1513/ : SUCCESS in 16s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-examples-zuul-docker/2130/ : FAILURE in 22m 31s

**Comment by gdgate** [comment]
Build failed (gate pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-storybook-zuul-docker/3237/ : SUCCESS in 19m 01s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-unit-tests-zuul-docker-el7/6244/ : FAILURE in 15m 21s
- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-merge-compliance-check/1515/ : SUCCESS in 25s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-examples-zuul-docker/2133/ : SUCCESS in 25m 11s

**Comment by gdgate** [comment]
Build succeeded (gate pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-storybook-zuul-docker/3238/ : SUCCESS in 23m 16s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-unit-tests-zuul-docker-el7/6245/ : SUCCESS in 10m 44s
- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-merge-compliance-check/1516/ : SUCCESS in 15s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-examples-zuul-docker/2134/ : SUCCESS in 25m 14s

**Pull request merged**